tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -78,6 +78,7 @@
             "echo $uuid > /var/lib/nova/compute_id"
 
         done
+      when: not platform_rhev|default(false)
```

Comment:
@archana203 could you check if this PR works on RHEV when we skip this step?

Reply:
@fao89 Yeah, it worked without this step.